dataprofiler/tests/profilers/profiler_options/test_base_inspector_options.py

```diff
@@ -47,3 +50,20 @@ def test_is_prop_enabled(self):
 
     def test_eq(self):
         super().test_eq()
+
+    def test_json_encode(self):
+        option = BaseInspectorOptions(is_enabled=False)
+
+        serialized = json.dumps(option, cls=ProfileEncoder)
+
+        expected_class = "BaseInspectorOptions"
+        expected_options_attributes = {"is_enabled"}
+        expected_is_enabled = option.is_enabled
+
+        actual_option_json = json.loads(serialized)
+
+        self.assertEqual(expected_class, actual_option_json["class"])
```

Comment:
In this case, if `"class"` doesn't exist, this will cause an exception that may not be clear.
One way to structure these instead that also validates the structure is to:
```python
actual_option_json = json.loads(serialized)
actual_option_json_data = actual_option_json.pop("data")

# validates that has the structure {"class": ...} after popping "data"
self.assertDictEqual({"class": "BaseInspectorOptions "}, actual_option_json)
self.assertEqual(
    expected_options_attributes, set(actual_option_json["data"].keys())
)
self.assertDictEqual({"is_enable": False}, actual_option_json.get("data"))
```
Same goes for error on last assert.